samples/conf.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Script server",
+  "port": "8000"
+//   "ssl": {
```

Comment:
Could you add some normal comment instead, please? 
And keep the sample configuration

Reply:
done